packages/@divvi/mobile/src/i18n/saga.test.ts

```diff
@@ -36,7 +43,7 @@ describe('i18n sagas', () => {
   })
 
   it('handles fetching over the air translations', async () => {
-    const translations = { someLang: { someNamespace: { someKey: 'someValue ' } } }
+    const translations = { someNamespace: { someKey: 'someValue ' } }
```

Comment:
not a strictly necessary change, but the language was never part of the response